app/src/ActionModal/components/ActionModal.vue

```diff
@@ -43,9 +43,12 @@
       <template v-if="!checkFeatureAvailable">
         <FeatureNotAvailable :feature="title" />
       </template>
-      <TmDataLoading
+      <div
```

Comment:
i think it would have been easier to just delete the contents of TmDataLoading and replace with 🤷 

`<img src="/img/spinner_blue@256.gif" class="spinner" />`

Reply:
OK, yes, I will change it to that